tests/php/InlineCssTest.php

```
@@ -0,0 +1,566 @@
+		// is_css_minified() can operate on it.
+		$source_dir  = '/tmp/wordpress/wp-content/themes/t';
+		$source_file = $source_dir . '/style.css';
+		// phpcs:ignore WordPress.WP.AlternativeFunctions.file_system_operations_file_put_contents
+		file_put_contents( $source_file, "body {\n\tcolor: red;\n}\n\nh1 {\n\tcolor: blue;\n}\n" );
+
+		$cached_file = '/tmp/wordpress/wp-content/cache/wppo/min/css/' . md5( 'third-party' ) . '.css';
+		// phpcs:ignore WordPress.WP.AlternativeFunctions.file_system_operations_file_put_contents
+		file_put_contents( $cached_file, 'body{color:red}' );
+		$cached_url = 'http://example.com/wp-content/cache/wppo/min/css/' . basename( $cached_file );
+
+		$minifier = \Mockery::mock( 'overload:PerformanceOptimise\Inc\Minify\CSS' );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Mockery' in method 'test_minify_css_minifies_third_party_path_data_handle'. (undefined)

(StaticAccess)